lib/samson/tasks.rb

```diff
@@ -0,0 +1,5 @@
+module Samson::Tasks
+  def self.start
+    LockCleaner.start
```

Comment:
Any reason for the indirection? Should this become a real registry class and keep instances?